Pegasus/include/Geometry.hpp

```diff
@@ -516,13 +531,16 @@ inline void Initialize<Sphere, Box>(SimpleShape const* a, SimpleShape const* b,
     auto box = static_cast<Box const *>(b);
     auto cache = static_cast<Cache<Sphere, Box>*>(cacheBase);
 
-    cache->sphereMassCenter = sphere->getCenterOfMass();
+    cache->sphereMassCenter = sphere->GetCenterOfMass();
     cache->sphereRadius = sphere->GetRadius();
 
-    cache->boxMassCenter = box->getCenterOfMass();
+    cache->boxMassCenter = box->GetCenterOfMass();
     box->GetAxes(cache->boxAxes[0], cache->boxAxes[1], cache->boxAxes[2]);
     CalculateBoxVertices(cache->boxAxes[0], cache->boxAxes[1], cache->boxAxes[2], cache->boxVertices.begin());
-    std::for_each(cache->boxVertices.begin(), cache->boxVertices.end(), [cache](auto& n) { n += cache->boxMassCenter; });
+    std::for_each(cache->boxVertices.begin(), cache->boxVertices.end(), [cache](auto& n)
```

Comment:
Nope, accumulate will calculate the sum of the vertices, but this code performs transformation by translation.